config/config-mp/src/test/java/io/helidon/config/mp/MpConfigTest.java

```diff
@@ -274,5 +275,29 @@ public static void reset() {
             counter.set(0);
         }
     }
+
+    @Test
+    void testEnvVar() {
+        ConfigProviderResolver instance = ConfigProviderResolver.instance();
+        ClassLoader myCl = Thread.currentThread().getContextClassLoader();
+        Config current = ConfigProvider.getConfig(myCl);
+
+        try {
+            instance.registerConfig(instance.getBuilder()
+                                            .withSources(MpConfigSources.environmentVariables())
+                                            .build(),
+                                    myCl);
+            Config myConfig = instance.getConfig(myCl);
+            // this must not throw an exception - path should be on any environment
+            // and the MP env var processing should make it available
+            String path = myConfig.getValue("path", String.class);
```

Comment:
Failing on windows `java.util.NoSuchElementException: Property "path" is not available in configuration`

Reply:
Because its `Path`

Reply:
Using an env var configured in pom.xml now.